app/code/Magento/Multishipping/Block/Checkout/Address/Select.php

```diff
@@ -28,11 +28,6 @@ class Select extends \Magento\Multishipping\Block\Checkout\AbstractMultishipping
      */
     protected $addressMapper;
 
-    /**
-     * @var bool
-     */
-    protected $_isScopePrivate = true;
```

Comment:
While it should be ok to remove the redeclarations of the inherited property with the same value, I don't think we should remove the `$_isScopePrivate = true` in this pull request (that should be done in 2.5 as part as deprecated property removal preparation)

Reply:
should i need revert all that ? only keep deprecate notes

Reply:
That is the only file where the removal should be reverted